composer.json

```
@@ -8,24 +8,28 @@
         {
             "type": "vcs",
             "url": "https://github.com/whilesmart/laravel-plugin-engine"
+        },
+        {
+            "type": "vcs",
+            "url": "https://github.com/whilesmart/laravel-user-authentication"
         }
     ],
     "require": {
-        "php": "^8.1",
-        "cviebrock/eloquent-sluggable": "^10.0",
+        "php": "^8.2",
+        "cviebrock/eloquent-sluggable": "^11.0",
         "guzzlehttp/guzzle": "^7.2",
-        "laravel/framework": "^10.10",
-        "laravel/sanctum": "^3.3",
+        "laravel/framework": "^11.45.1",
+        "laravel/sanctum": "^4.0",
         "laravel/tinker": "^2.8",
         "zircote/swagger-php": "^5.0",
-        "whilesmart/laravel-plugin-engine": "@dev"
-    },
+        "whilesmart/laravel-user-authentication": "main-dev"
```

```suggestion
        "whilesmart/laravel-user-authentication": "main-dev"
```

I guess we have done releases for this ? If so let's use a specific version.